.pre-commit-config.yaml

```diff
@@ -87,7 +87,7 @@ repos:
         exclude: .pre-commit-config.yaml
 
   - repo: https://github.com/codespell-project/codespell
-    rev: v2.2.6
+    rev: v2.3.0
     hooks:
       - id: codespell
-        args: ["-L", "atleast,ans,doub,inout"]
+        args: ["-L", "atleast,ans,doub,inout,AtMost"]
```

Comment:
Oh, did this bug get fixed in 2.3.0? It used to be you had to put all lower case versions here.

Reply:
Didn't know there was a bug, but this resolved the pre-commit issue